core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -182,7 +183,7 @@ class Translations : LinkedHashMap<String, TranslationEntry>() {
 
         DiacriticSupport.reset()
         for (language in getLanguagesWithTranslationFile()) {
-            tryReadTranslationForLanguage(language)
+            tryReadTranslationForLanguage(language, noDiacritics = true)
```

Comment:
Please add a comment here
"We load the translations with no diacritics for TranslationFileWriter, so what we read will be what we write"

Also, are `DiacriticSupport.reset()` and `DiacriticSupport.freeTranslationData()` lines, above and below, still necessary?